pkg/resource/secret_reconciler.go

```diff
@@ -33,14 +34,9 @@ import (
 // propagated to the named resource of the same kind, assuming it exists and
 // consents to propagation.
 const (
-	AnnotationKeyPropagateToNamespace = "crossplane.io/propagate-to-namespace"
-	AnnotationKeyPropagateToName      = "crossplane.io/propagate-to-name"
-	AnnotationKeyPropagateToUID       = "crossplane.io/propagate-to-uid"
-)
+	AnnotationKeyPropagateToPrefix = "to.propagate.crossplane.io"
+	SlashDelimeter                 = "/"
 
-// Supported resources with all of these annotations consent to be fully or
-// partially propagated from the named resource of the same kind.
-const (
 	AnnotationKeyPropagateFromNamespace = "crossplane.io/propagate-from-namespace"
 	AnnotationKeyPropagateFromName      = "crossplane.io/propagate-from-name"
 	AnnotationKeyPropagateFromUID       = "crossplane.io/propagate-from-uid"
```

Comment:
Could we make these `propagate.crossplane.io/from-namespace`? This way both ends of our propagation annotations are within the `propagate.crossplane.io` annotation domain.

Reply:
Ah yes definitely. Given that we have `to.propagate.crossplane.io` should we do `from.propagate.crossplane.io/namespace`, etc.?

Reply:
Yeah, that seems sensible.

Reply:
Awesome, done!